### ISSUE TITLE
BigIntegerField confusion

### ISSUE THREAD

**Issue by etam** [golemfactory/clay#2169]
In our `model.py`, `BigIntegerField` is derived from `CharField`. But there is also a `BigIntegerField` class defined in `peewee`, which is directly mapped to `bigint` sql type. In `Income` type, `value` is our `BigIntegerField`, but in sqlite database file, the column type is bigint (somehow it has no problem with storing strings).

For some unknown reason, probably caused by inconsistency between database and private key being recreated by #1480, I have incomes in database with value 0.

So this happens periodically, while GUI is connected:
```
ERROR    [twisted                            ] TypeError: int() can't convert non-string with explicit base: Traceback (most recent 
call last):                                                                                                                        
  File "/home/etam/code/github/golemfactory/golem/log_hardware/venv/lib/python3.6/site-packages/autobahn/twisted/websocket.py", line 162, in _onMessage
    self.onMessage(payload, isBinary)
  File "/home/etam/code/github/golemfactory/golem/log_hardware/venv/lib/python3.6/site-packages/autobahn/wamp/websocket.py", line 95, in onMessage
    self._session.onMessage(msg)
  File "/home/etam/code/github/golemfactory/golem/log_hardware/venv/lib/python3.6/site-packages/autobahn/wamp/protocol.py", line 895, in onMessage
    on_reply = txaio.as_future(endpoint.fn, *invoke_args, **invoke_kwargs)
  File "/home/etam/code/github/golemfactory/golem/log_hardware/venv/lib/python3.6/site-packages/txaio/tx.py", line 417, in as_future
    return maybeDeferred(fun, *args, **kwargs)
--- <exception caught here> ---
  File "/home/etam/code/github/golemfactory/golem/log_hardware/venv/lib/python3.6/site-packages/twisted/internet/defer.py", line 150, in maybeDeferred
    result = f(*args, **kw)
  File "/home/etam/code/github/golemfactory/golem/log_hardware/golem/client.py", line 797, in get_incomes_list
    return self.transaction_system.get_incoming_payments()
  File "/home/etam/code/github/golemfactory/golem/log_hardware/golem/transactions/transactionsystem.py", line 81, in get_incoming_payments
    return [item(income) for income in incomes]
  File "/home/etam/code/github/golemfactory/golem/log_hardware/golem/transactions/transactionsystem.py", line 81, in <listcomp>
    return [item(income) for income in incomes]
  File "/home/etam/code/github/golemfactory/golem/log_hardware/venv/lib/python3.6/site-packages/peewee.py", line 2289, in next
    obj = self.qrw.iterate()
  File "/home/etam/code/github/golemfactory/golem/log_hardware/venv/lib/python3.6/site-packages/peewee.py", line 2349, in iterate
    return self.process_row(row)
  File "/home/etam/code/github/golemfactory/golem/log_hardware/venv/lib/python3.6/site-packages/peewee.py", line 2432, in process_row
    setattr(instance, column, f(row[i]) if f is not None else row[i])
  File "/home/etam/code/github/golemfactory/golem/log_hardware/golem/model.py", line 75, in python_value
    return int(value, 16)
builtins.TypeError: int() can't convert non-string with explicit base
```

**Comment by etam** [comment]
@mfranciszkiewicz @Krigpl

**Comment by etam** [comment]
Oh, so it was already fixed by #2162
